src/transformers/convert_slow_tokenizer.py

```diff
@@ -1552,6 +1552,35 @@ def converted(self) -> Tokenizer:
         return tokenizer
 
 
+class LlamaTikTokenConverter(TikTokenConverter):
```

Comment:
Not sure Llama should be different, let's try to have the general `TikTokenConverter`